integration_tests/utils.py

```diff
@@ -441,3 +441,83 @@ def call_contact(self, w3):
         assert "world" == self.contract.functions.greet().call()
 
         return send_store_tx
+
+
+class TestRevert:
```

Comment:
It seems many codes can be reused with `class Greeter`.

Reply:
Yes. This can be improved...